### ISSUE TITLE
Allow normal whitespace when reading EditorConfig settings file.

### ISSUE THREAD

**Issue by bfinney-thoughtworks** [editorconfig/editorconfig-emacs#161]
Text files can contain various normal whitespace (line breaks, horizontal space, vertical space, etc.) and remain compatible with the EditorConfig file format.

Here is an example file (rename to `.editorconfig`):

[example-editorconfig.txt](https://github.com/editorconfig/editorconfig-emacs/files/1978142/example-editorconfig.txt)

That file, named as `.editorconfig`, works correctly with most EditorConfig libraries, but fails with ‘editorconfig-emacs’:

```
Error (editorconfig): Error while reading config file: /Users/bfinney/Projects/ECMAScript/JavaScript30/devel/.editorconfig:29:
    
.  Styles will not be applied.
```

For reference, the EditorConfig [file format details documentation](http://editorconfig.org/#file-format-details) says the format is based on Python's `configparser` format. Using Python, the same file parses correctly:

```python
>>> import configparser
>>> parser = configparser.SafeConfigParser()
>>> parser.read('.editorconfig')
['…/.editorconfig']
>>> parser.get('*', 'indent_size')
'4'
```

Probably the correct way to resolve this is to allow any Unicode whitespace character ([those with the `WSpace=Y` property](https://en.wikipedia.org/wiki/Whitespace_character#Unicode)) when reading the settings file.

**Comment by 10sr** [comment]
Thanks for reporting!
It should be fixed now, try new version 🙏

**Comment by bfinney-thoughtworks** [comment]
On Tue, May 8, 2018 at 12:52 PM, 10sr <notifications@github.com> wrote:

> Thanks for reporting!
> It should be fixed now, try new version 🙏
>
Thank you for making this change. I don't see a new version at
https://github.com/editorconfig/editorconfig-emacs/releases so I guess I
need to wait for that?

-- 
Ben Finney
ThoughtWorks Australia
bfinney@thoughtworks.com

**Comment by jgkamat** [comment]
If you pull from melpa you should see an update in the next couple of hours. If you pull from melpa stable you'll need to wait for the next stable release...

**Comment by bfinney-thoughtworks** [comment]
> If you pull from melpa stable you'll need to wait for the next stable release

Thanks. What needs to be done before a new stable release of editorconfig-emacs then?

**Comment by jgkamat** [comment]
> Thanks. What needs to be done before a new stable release of
> editorconfig-emacs then?

That's mainly whenever the maintainers feel that a stable and maintainable
point has been reached in development. You should be able to get it from melpa
unstable (by pinning the package) if you want to try out the unstable version.

**Comment by 10sr** [comment]
I did not notice we have not had a release recently XP
I just published v0.7.12, which will be able to get from melpa stable shortly!
